learn/tooling-guide/cli-tools/update-tool.md

```diff
@@ -30,28 +30,6 @@ Ballerina distributions are released via two different release channels at the m
 
 Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
 
-#### Versioning of the releases
```

Comment:
Its ok to remove this as now we don't follow semver versions. 
Shall we mention that swan lake release channel is the primary release and mention that 1.2.x release will be done on maintenance mode with patches?  
@parkavi11 @praneesha

Reply:
+1

Reply:
we should not mention that 1.2.x is on maintenance mode specifically